modern/base/Dockerfile

```diff
@@ -91,6 +91,7 @@ RUN apt-get -qq update \
     && apt-get -qq update \
     && rm -rf /var/lib/apt/lists/* \
     && rm -rf /tmp/* \
+    && CONAN_USER_HOME=/opt/conan conan download binutils/2.37@ -r conancenter \
```

Comment:
So we don't forget later (in case we opt for this alternative): choose one recipe-revision, please (does it have requirements? do we want to use lockfile?)

Also, maybe it is worth considering the `-g deploy` to that folder (easier to add to PATH) and removing the cache afterward.

Reply:
I think you are considering Conan Install instead, right? Yes, it sounds safer, as we can lock a revision.

Reply:
we can lock RREV and PREV to be sure (alternatively, use `conan create` against git revision, but it's slower)

Reply:
bintutils requires more dependencies, I think lockfile fits better in this case, so we can point all packages and their revisions.